common.mk

```diff
@@ -12,3 +12,10 @@ ZIPINPUT = true
 # libgsl-dev, libboost-all-dev, libsuitesparse-dev, liblpsolve55-dev, libmysql++-dev and libsqlite3-dev 
 # are not available
 COMPGENEPRED = true
+
+# set TESTING to false if you don't need testing for the comparative gene prediction mode (CGP) or
+# the required libraries 
+# libboost-serialization-dev 
+# are not available
+TESTING = true
```

Comment:
Hold on, until we have an answer from Giovanna, whether this default is really not a large risk in master (installation, runtime). Possibly we set this as developers only.

Reply:
@mglgnn and I talked. 
Please move the TESTING = true to src/Makefile, as users are asked to edit common.mk but not src/Makefile.
After the pull is done and the testing branch merged, Giovanna will use the very test to confirm that some upcoming refactoring that regards TESTING is not braking anything in both settings TESTING=true/false.

Reply:
Moved TESTING to src/Makefile.